_studio/shared/umc/codec/h264_dec/include/umc_h264_task_supplier.h

```diff
@@ -728,12 +728,9 @@ inline int32_t CalculateDPBSize(uint8_t & level_idc, int32_t width, int32_t heig
         case H264VideoDecoderParams::H264_LEVEL_6:
         case H264VideoDecoderParams::H264_LEVEL_61:
         case H264VideoDecoderParams::H264_LEVEL_62:
-            MaxDPBMbs = 696320;
-            break;
+            // fall-through
```

Comment:
Seems issue could be also fixed w/o change this line.

Reply:
Suggest not to change this line, otherwise it will break AVC Level6+ support feature.

Reply:
@guowillyy , why will it break 6+?  Both cases H264VideoDecoderParams::H264_LEVEL_6* and default result in the same MaxDPBMbs = 696320.

Reply:
Right, it will not break 6+ as @dmitryermilov pointed out.  But I will remove this change anyway, since it's unnecessary (gratuitous).